core/frontend/src/components/vehiclesetup/PwmSetup.vue

```diff
@@ -458,15 +468,15 @@ export default Vue.extend({
       clearInterval(this.motor_zeroer_interval)
     },
     arm() {
-      this.armDisarm(true, true)
+      armDisarm(true, true)
       this.arming_timeout = setTimeout(() => {
         if (this.desired_armed_state === this.is_armed) return
         this.desired_armed_state = this.is_armed
         console.warn('Arming failed!')
       }, 5000)
     },
     disarm() {
-      this.armDisarm(false, true)
+      armDisarm(false, true)
```

Comment:
Missing await here, right ?

Reply:
we dont _have_ to have an await there, unlike python, js will run the code regardless. as (now) there's nothing after it, I don't see the need. it would probably require a bunch of ther asyncs around the code. but I can do it if you want.

I did realise there was duplicated logic and removed the timeout, though, as we do have a timeout in the helper functions